driver/bpf/filler_helpers.h

```diff
@@ -19,6 +19,11 @@ or GPL2.txt for full copies of the license.
 
 #include "../ppm_flag_helpers.h"
 
+static inline bool in_port_range(uint16_t port, uint16_t min, uint16_t max)
+{
+    return port >= min && port <= max;
```

Comment:
Please consider using the proper indentation: the kernel code is indented with tabs at width 8, whereas this line is indented with 4 spaces. There is also a `checkpatch.pl` in the kernel tree that points these things out.